examples/evolve_dstream_surf.py

```diff
@@ -288,13 +311,17 @@ def _get_df_save_handler(path: str) -> typing.Callable:
     # do simulation
     common_ancestor = Organism.create_founder()
     init_population = [common_ancestor.CreateOffspring() for _ in range(100)]
-    end_population = evolve_drift(init_population)
+    end_population = evolve_drift(init_population, fossil_interval=args.fossil_interval)
 
     # mark non-tip taxa extinct
     del common_ancestor
     del init_population
     gc.collect()
 
+    print(syst.get_total_orgs())
+    print(len(end_population))
```

Comment:
```suggestion
    print("final population size:", len(end_population))
```
Can we also calculate/print the number of fossils included?